.github/workflows/CompatHelper.yml

```
@@ -1,16 +1,45 @@
 name: CompatHelper
 on:
   schedule:
-    - cron: 19 22 * * *
+    - cron: 0 0 * * *
```

I prefer having the cron times be non-standard, as a habit. 
```suggestion
    - cron: 19 22 * * *
```